tests/contrib/django/test_django.py

```diff
@@ -1333,6 +1334,21 @@ def test_middleware_trace_request_ot(client, test_spans, tracer):
     assert sp_request.get_tag("http.method") == "GET"
 
 
+def test_set_request_tags_handles_improperly_configured_error(client):
+    """
+    Since it's difficult to reproduce the ImproperlyConfigured error via django (server setup), will instead
+    mimic the failure by mocking the user_is_authenticated to raise an error.
+    """
+    # patch django._patch - django.__init__.py imports patch.py module as _patch
+    with mock.patch(
+        "ddtrace.contrib.django._patch.user_is_authenticated", side_effect=django.core.exceptions.ImproperlyConfigured
+    ):
+        try:
+            client.get("/")
```

Comment:
```suggestion
        with pytest.raises(django.core.exceptions.ImproperlyConfigured
):
            client.get("/")
```

Reply:
We should also check to make sure that the trace was still correctly created and finished (wasn't impacted by the error)